# try:
#     print(출력되나)
#     # 아래 같은 구문 에러는 못 잡음
#     # print(출력 되나)
# except:
#     print("절대 안 되지")
#
# exception

try:
    print(param)

# 발생하는 오류 종류가 exception이면 e에 저장해서 print
except Exception as e:
    print(e)

# print(paa)
